Doc/library/sqlite3.rst

```diff
@@ -25,6 +25,15 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+This document includes four main sections:
+
+* :ref:`sqlite3-tutorial`: teaches how to use the sqlite3 module
+* :ref:`sqlite3-reference`: describes the classes and methods defined in the
+  module
+* :ref:`sqlite3-howtos`: explains how to solve specific problems
+* :ref:`sqlite3-concepts`: provides more background and insights about the
+  design of the module
```

Comment:
```suggestion
* :ref:`sqlite3-concepts` provides more background and insights
  about the module's design
```

* Reword to flow better and be a bit more concise
* Rephrase as a sentence

Reply:
Actually, I'd like to reword this completely, because it does _not_ talk about the module's design. The only item in "Concepts" is the transaction control explanation. The next upcoming item is the new PEP 249 autocommit transaction control explanation. So, two explanations. No concepts 😆

Reply:
PTAL